incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -0,0 +1,52 @@
+image:
+  repository: ghcr.io/truecharts/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "v4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+
+
+env:
+  HA_URL: "http://192.168.5.31:8123"
+  # used to point appdaemon to your home assistant instance
+  TOKEN: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJjNjNiMzZkM2QwMTg0M2Y4OGQ2Y2JkYjY4MmViNGQ2NiIsImlhdCI6MTYxODk0MDc4OSwiZXhwIjoxOTM0MzAwNzg5fQ.CUXm9K4jSS--qBQZ9DahqFL1YIgIcCK32pfORqiQ0ns"
+  # used to authenticate app daemon to your home assistant instance
+  DASH_URL: "localhost:5050"
+  # url for the appdaemon dashboard, an alternative interface for home assistant
+  TZ: "America/Chicago"
```

Comment:
You cannot set this!
As clearly stated in the manual we already set this based on our TZ env-var, you are now creating duplicates!

Reply:
The application was crashing prior to adding the TZ variable to the test-values.yaml file. What is the best way to handle this during the test?

Reply:
using TZ not env.tz

Reply:
@Ornias1993 I'm guessing this is simple but I'm struggling with the TZ vs env.tz. Would you be able to provide an example block of code?

Reply:
I think its important you read into how YAML code is structured, instead of me providing an example ;-)